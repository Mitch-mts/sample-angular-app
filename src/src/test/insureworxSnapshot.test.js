import renderer from 'react-test-renderer';
import Link from '../Link';

it('renders correctly', () => {
  const tree = renderer
    .create(<Link page="http://localhost:4200/">InsureworX</Link>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});