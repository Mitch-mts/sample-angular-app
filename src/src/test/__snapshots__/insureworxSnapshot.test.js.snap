// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<a
  className="normal"
  href="http://localhost:4200/"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  InsureworX
</a>
`;
